cli/install/install.go

```diff
@@ -0,0 +1,572 @@
+		return nil
+	}
+	defer distoFile.Close()
+
+	scanner := bufio.NewScanner(distoFile)
+	for scanner.Scan() {
+		if m := PrettyName.FindStringSubmatch(scanner.Text()); m != nil {
+			distroInfo.Name = strings.Trim(m[1], `"`)
+		} else if m := ID.FindStringSubmatch(scanner.Text()); m != nil {
+			distroInfo.Vendor = strings.Trim(m[1], `"`)
+		} else if m := VersionID.FindStringSubmatch(scanner.Text()); m != nil {
+			distroInfo.Version = strings.Trim(m[1], `"`)
+		}
+	}
+	return &distroInfo
+}
+
+func DetectOsName() string {
+	if runtime.GOOS == "darwin" {
+		return "darwin"
+	}
+	if runtime.GOOS == "windows" {
+		return "windows"
+	}
+	if runtime.GOOS == "linux" {
+		distroInfo := GetDistroName()
+		return distroInfo.Name
+	}
+	return ""
+}
+
+// isDeprecated checks wethere version of programm is below 1.10.0.
+func isDeprecated(version string) bool {
+	splitedVersion := strings.Split(version, ".")
+	if len(splitedVersion) < 2 {
+		return false
+	}
+	if splitedVersion[0] == "1" && len(splitedVersion[1]) < 2 {
+		return true
+	}
+	return false
+}
+
+// getLatestTarantoolTag returns latest tag from tarantool repository.
+func getLatestTarantoolTag() (string, error) {
+	cmd := exec.Command("git", "ls-remote", "--tags", "--sort="+
+		"v:refname", "https://github.com/tarantool/tarantool.git")
+	readPipe, writePipe, _ := os.Pipe()
+	cmd.Stdout = writePipe
+	cmd.Stderr = os.Stderr
+	cmd.Start()
+	err := cmd.Wait()
+	writePipe.Close()
+	if err != nil {
+		return "", err
+	}
+
+	var buf bytes.Buffer
+	io.Copy(&buf, readPipe)
+	versions := buf.String()
+	versionsArray := strings.Split(versions, "\n")
+	latestTag := versionsArray[len(versionsArray)-2]
+	latestTag = strings.TrimSuffix(latestTag, "^{}")
+	trimpos := strings.LastIndex(latestTag, "/") + 1
+	latestTag = latestTag[trimpos:]
+	return latestTag, nil
+}
+
+// copy copies file from source to destination.
+func copy(src string, dst string) error {
+	// Read all content of src to data.
+	data, err := ioutil.ReadFile(src)
+	if err != nil {
+		return err
+	}
+	// Write data to dst.
+	err = ioutil.WriteFile(dst, data, 0777)
+	return err
+}
+
+func copyDir(src string, dst string) error {
+	cmd := exec.Command("cp", "--recursive", src, dst)
+	cmd.Start()
+	err := cmd.Wait()
+	return err
+}
+
+// checkDependecie checks if programm is installed.
+func checkDependecie(programm string) bool {
+	cmd := exec.Command(programm, "--version")
+	cmd.Start()
+	err := cmd.Wait()
+	return err == nil
+}
+
+// checkPythonDependencie checks if python packet is installed.
+func checkPythonDependencie(packet string) bool {
+	cmd := exec.Command("pip3", "list")
+	readPipe, writePipe, _ := os.Pipe()
+	cmd.Stdout = writePipe
+	cmd.Stderr = os.Stderr
+	cmd.Start()
+	cmd.Wait()
+	writePipe.Close()
+	var buf bytes.Buffer
+	io.Copy(&buf, readPipe)
+	packets := buf.String()
+	return strings.Contains(packets, packet)
+}
+
+func CheckLibDependencie(library string) bool {
+	cmd := exec.Command("bash", "-c", "ls"+" /lib/*/")
```

Comment:
Why not using pkg manager for Debian/Ubuntu?

Reply:
used dpkg for Debian/Ubuntu